client/packages/common/src/ui/components/popover/DisabledNotification/DisabledNotification.stories.tsx

```diff
@@ -3,25 +3,36 @@ import { StoryFn } from '@storybook/react';
 import Box from '@mui/material/Box';
 import { BasePopover } from '../BasePopover';
 import { BaseButton } from '../../buttons';
-import { useDisabledNotification } from './useDisabledNotification';
+import { useDisabledNotificationPopover } from './useDisabledNotificationPopover';
+import { useDisabledNotificationToast } from './useDisabledNotificationToast';
 
 export default {
   title: 'Popover/DisabledNotification',
   component: BasePopover,
 };
 
 const Example: StoryFn = () => {
-  const { show, DisabledNotification } = useDisabledNotification({
+  const { show, DisabledNotification } = useDisabledNotificationPopover({
     title: 'Permission denied',
     message: 'You do not have permission to perform this action.',
   });
+
+  const showToast = useDisabledNotificationToast(
+    'You do not have permission to perform this action.'
+  );
+
   return (
     <>
       <DisabledNotification />
       <Box>
         <BaseButton onClick={show}>Click this button.. if you dare!</BaseButton>
         <div>You can click the message to close it.</div>
       </Box>
+      <Box>
+        <BaseButton onClick={showToast}>
+          For a toast version, click this one
```

Comment:
I'm confused about the purpose of these untranslated strings, but I am guessing that it isn't part of the public facing codebase anyhow

Reply:
yes - I don't bother translating the text in the storybook stories; it's only really us that uses storybook - not intended for system users.